build/files.js

```diff
@@ -98,10 +98,47 @@ const headRegexp = /(^module.exports = \w+;?)/m
         + 'if (!EE.listenerCount) EE.listenerCount = function(emitter, type) {\n'
         + '  return emitter.listeners(type).length;\n'
         + '};\n/*</replacement>*/\n'
+      ]
```

Comment:
@dcousens the changes to this file is the meat of the update, most of the other changes are consequences of this